tests/devices/test_default_qutrit.py

```diff
@@ -1391,3 +1396,494 @@ def test_density_matrix_all_wires(self, qutrit_device_2_wires, wires, expected):
         qutrit_device_2_wires.apply(ops)
 
         assert np.allclose(qutrit_device_2_wires.density_matrix(wires), expected)
+
+
+# JAX
+# ======================================================================================================================
+pytestmark = pytest.mark.jax
+jax = pytest.importorskip("jax")
+jnp = pytest.importorskip("jax.numpy")
+config = pytest.importorskip("jax.config")
+
+
```

Comment:
```suggestion
@pytest.mark.jax
```

Can remove separate definition in line 1403 above; please do for the TF/Torch versions as well

Reply:
Done